helpers.go

```diff
@@ -514,3 +521,52 @@ func askForConfirmation(s string) bool {
 		}
 	}
 }
+
+type Cache struct {
+	Timestamp string   `json:"timestamp"`
+	Versions  []string `json:"versions"`
+}
+
+func (gb *GoBrew) getVersionsFromCache() []string {
+	cacheFile := filepath.Join(gb.installDir, "cache.json")
+	if _, err := os.Stat(cacheFile); err == nil {
+		data, e := os.ReadFile(cacheFile)
+		if e != nil {
+			return []string{}
+		}
+
+		var cache Cache
+		if e = json.Unmarshal(data, &cache); e != nil {
+			return []string{}
+		}
+
+		timestamp, e := time.Parse(time.RFC3339, cache.Timestamp)
+		if e != nil {
+			return []string{}
+		}
+
+		// cache for 20 minutes
+		if time.Now().UTC().After(timestamp.Add(20 * time.Minute)) {
```

Comment:
Just a thought. May be an option as `--no-cache` on cli to not use cache might do some good?
Or an option `--clean-cache` to bust the current cache.
It is 20 mins so no problems but might be a good compliance addition.

Reply:
Yes, it will need to be added. There is also an idea to configure the cache time. Leave 20 minutes by default, but with the ability to set any value.

I created a separate task so I wouldn't forget: #202